frontend/src/App.tsx

```diff
@@ -22,36 +22,39 @@ function App(): ReactElement {
   const dispatch = global.dispatch;
 
   useEffect(() => {
-    //establish websocket connection
-    const websocket: WebSocket = new WebSocket(
-      `${process.env.REACT_APP_WEBSOCKET_URL}`
-    );
-
-    //when established, join the room as the professor
-    websocket.onopen = () => {
-      websocket.send(
-        JSON.stringify({
-          action: "professorJoinRoom",
-          courseId: state.courseId,
-        })
+    if (state.courseId) {
+      console.log(state.courseId);
```

Comment:
remove console log

Reply:
You're looking at an old commit, that's already removed